packages/app/src/app/pages/Sandbox/Editor/Workspace/Advertisement/index.ts

```diff
@@ -0,0 +1 @@
+export { Advertisement } from './Advertisement';
```

Comment:
@Saeris Why cant we just delete this file and rename `Advertisement.jsx` to `index.tsx`? 🤔

Right now we're creating every time an extra file for no reason at all imo.